src/meetings/2021-11-22.md

```diff
@@ -0,0 +1,31 @@
+# 2021-11-22
+
+| Info                                           |                          |
+| ---------------------------------------------- | ------------------------ |
+| Time                                           | [See in your time zone]  |
+| Calendar event (with Zoom details)             | [link][cal]              |
+| Social hour calendar event (with Zoom details) | [link][calsh]            |
+| Recording                                      | (link posted after call) |
+
+[see in your time zone]: https://everytimezone.com/s/91c9791f
+[cal]: https://calendar.google.com/event?action=TEMPLATE&tmeid=NnVibTVzcmEwdnZxMTBtYjZ0ZmZxaWtyYXQgN24wdnZvcWZlMGtibms2aTA0dWl1NTJ0MzBAZw&tmsrc=7n0vvoqfe0kbnk6i04uiu52t30%40group.calendar.google.com
+[calsh]: https://calendar.google.com/event?action=TEMPLATE&tmeid=NGd2N2FzYjBnMHFiNXA4dWdtYWZ2ZmxobmwgN24wdnZvcWZlMGtibms2aTA0dWl1NTJ0MzBAZw&tmsrc=7n0vvoqfe0kbnk6i04uiu52t30%40group.calendar.google.com
+
+## Agenda
+
+The theme for this meeting is Rust in the Linux kernel.
```

Comment:
@nikomatsakis is there anything else we could add to the "theme" to describe this CTCFT a bit better?

Reply:
```suggestion
This meeting we've invited some of the people being the work to integrate Rust in the Linux kernel to come and speak. We've asked them to give us a feel for how the integration works and help identify those places where the experience is rough. The expectation is that we can use this feedback as an input when deciding what work to pursue and what features to prioritize for stabilization.
```

Reply:
Maybe like that?

Reply:
This meeting we've invited some of the people working to integrate Rust into the Linux kernel to come and speak. We've asked them to give us a feel for how the integration works and help identify those places where the experience is rough. The expectation is that we can use this feedback as an input when deciding what work to pursue and what features to prioritize for stabilization.

Reply:
```suggestion
This meeting we've invited some of the people working to integrate Rust into the Linux kernel to come and speak. We've asked them to give us a feel for how the integration works and help identify those places where the experience is rough. The expectation is that we can use this feedback as an input when deciding what work to pursue and what features to prioritize for stabilization.
```